app/customer/customer_class.py

```diff
@@ -0,0 +1,95 @@
+from app.car.car_class import Car
+
+
+class Customer:
+    def __init__(self, customer: dict, fuel_price: str) -> None:
+        self.name = customer["name"]
+        self.product_cart = customer["product_cart"]
+        self.location = customer["location"]
+        self.money = customer["money"]
+        self.car = Car(customer["car"], float(fuel_price))
+
+    def go_to_shop(self, shops: dict) -> None:
+        print(f"{self.name} has {self.money} dollars")
+
+        cheapest_store = self.find_cheapest_store(shops)
+
+        if cheapest_store is not None:
+            self.detailed_store_transactions(cheapest_store)
+
+    def cost_of_way(self, shop_location: list) -> float:
```

Comment:
add additional typehint for list inner objects like list[YOUR_VALUE_TYPE]
for example: list that contains string elements has typehint list[str]